spring-cloud-gcp-examples/spring-cloud-gcp-trace-example/src/main/java/com/example/Application.java

```diff
@@ -0,0 +1,41 @@
+/*
+ *  Copyright 2017 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.example;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+/**
+* Sample spring boot application.
+*
+* @author Ray Tsang
+*/
+@SpringBootApplication
+@Configuration
```

Comment:
`@SpringBootApplication` is `@Configuration` per se. So, we don't need this extra annotation here

Reply:
thnx! removed.